src/i18n/locale-en.json

```diff
@@ -147,6 +148,30 @@
         "remove_nodes_dialog": {
             "title": "Remove nodes from cluster",
             "content": "Select nodes to be removed from existing cluster"
+        },
+        "update_cluster_group_dialog": {
+            "title": "Update cluster group",
```

Comment:
You also need to update test-cypress/fixtures/locale-en.json.

Reply:
done